misc/klayout/build.sh

```diff
@@ -7,18 +7,34 @@ set -x
 UNAME_OUT="$(uname -s)"
 case "${UNAME_OUT}" in
     Linux*)     OS=Linux;;
+    Darwin*)    OS=Mac;;
     *)          OS="${UNAME_OUT}"
                 echo "Unknown OS: ${OS}"
                 exit;;
 esac
 
 cd ${SRC_DIR}
-./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
+if [[ $OS == "Linux" ]]; then
+    ./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
+    bin_ext=""
+    lib_ext=".so"
+elif [[ $OS == "Mac" ]]; then
+    # Support `any_cast` workaround: https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+    ./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
```

Comment:
Maybe we can move this out of the if because it's the same on both platform?

Reply:
I thought these commands might need to be different, but surprisingly the same command actually works!

I'll combine the two commands.